src/sources/objectionjs/utils/base-model.ts

```diff
@@ -0,0 +1,30 @@
+import addFormats from 'ajv-formats';
+import { AjvValidator, Model } from 'objection';
+
+// https://github.com/Vincit/objection.js/issues/2249#issuecomment-1075898552
```

Comment:
If this is valid would be worth to add some comment, what about is this comment

Reply:
Added thanks 😄